torch/distributed/distributed_c10d.py

```diff
@@ -4774,6 +4775,11 @@ def new_group(
             in that non-member ranks don't need to call into API and don't
             join the barrier.
         group_desc (str, optional): a string to describe the process group.
+        device_id (torch.device, optional): a single, specific device
+            to "bind" this process to, allowing for backend-specific
+            optimizations.  only under NCCL: the communicator is immediately formed
+            (calling``ncclCommInit*`` immediately rather than the normal lazy
+            call)
```

Comment:
This piece of documentation seems to come from `init_process_group`. I wonder if we could update them now that our view about `device_id` is clearer? For example:
```
a single, specific device to "bind" this process to. The `new_group` call will try to initialize a communication backend for the device if this field is given.
```